webapp/channels/src/components/browse_channels/browse_channels.tsx

```diff
@@ -3,30 +3,30 @@
 
 import classNames from 'classnames';
 import React from 'react';
-import { useSelector } from 'react-redux';
-import { FormattedMessage } from 'react-intl';
+import {useSelector} from 'react-redux';
+import {FormattedMessage} from 'react-intl';
 
-import { GenericModal } from '@mattermost/components';
-import type { Channel, ChannelMembership, ChannelSearchOpts, ChannelsWithTotalCount } from '@mattermost/types/channels';
-import { DisplaySettings } from "@mattermost/types/config";
-import { GlobalState } from '@mattermost/types/store';
-import type { RelationOneToOne } from '@mattermost/types/utilities';
+import {GenericModal} from '@mattermost/components';
+import type {Channel, ChannelMembership, ChannelSearchOpts, ChannelsWithTotalCount} from '@mattermost/types/channels';
+import {DisplaySettings} from "@mattermost/types/config";
+import {GlobalState} from '@mattermost/types/store';
+import type {RelationOneToOne} from '@mattermost/types/utilities';
 
 import Permissions from 'mattermost-redux/constants/permissions';
-import type { ActionResult } from 'mattermost-redux/types/actions';
+import type {ActionResult} from 'mattermost-redux/types/actions';
 
 import LoadingScreen from 'components/loading_screen';
 import NewChannelModal from 'components/new_channel_modal/new_channel_modal';
 import TeamPermissionGate from 'components/permissions_gates/team_permission_gate';
 import SearchableChannelList from 'components/searchable_channel_list';
 
-import { getHistory } from 'utils/browser_history';
-import Constants, { ModalIdentifiers, RHSStates, StoragePrefixes } from 'utils/constants';
-import { getRelativeChannelURL } from 'utils/url';
-import { localizeMessage } from 'utils/utils';
```

Comment:
can you remove these unrelated changes, its making review a bit difficult due these extra changes

Reply:
Done that already.